test/k8s/hubble.go

```diff
@@ -25,9 +25,9 @@ import (
 var _ = Describe("K8sAgentHubbleTest", func() {
 	// We want to run Hubble tests both with and without our kube-proxy
 	// replacement, as the trace events depend on it. We thus run the tests
-	// on GKE and our 4.9 pipeline.
+	// on GKE and our 4.19 pipeline.
 	SkipContextIf(func() bool {
-		return helpers.RunsOn419OrLaterKernel() || helpers.RunsOnAKS()
+		return helpers.RunsOnNetNextKernel() || helpers.RunsOnAKS()
```

Comment:
Shouldn't this use `RunsWithKubeProxyReplacement()` to satisfy the comment? Then we don't need to keep updating which kernels to run here or the comment.

Reply:
Actually now that 4.19 is the minimum, can we reuse `RunsWithKubeProxyReplacement()` more widely for other cases that are updated in this PR? It would be nice if we didn't have so many kernel version dependencies scattered throughout the test code.

Reply:
I thought about doing the `helpers.Runs*()` refactoring so that the skip conditions / configuration choices were more explicit and consistent.

However, I've started considering `test/k8s/*` to be a dead code which should be touched as little as possible, and which will be removed immediately after we have finished extending the coverage of `ci-datapath`.

Reply:
The first time someone told me to stop investing time into the ginkgo suite was 2 years ago (shortly after the CLI came out). I'm glad I didn't listen.